tests/scripts/run-xunit-perf.py

```diff
@@ -0,0 +1,466 @@
+            warmupRun,
+            '--append']
+
+    filename = os.path.join(benchmarkOutputDir, lvRunId + '-' + benchname + '.xml')
+
+    run_args = [python, os.path.join(benchviewPath, 'measurement.py')] + lvMeasurementArgs + [filename]
+    log('')
+    log(" ".join(run_args))
+
+    proc = subprocess.Popen(run_args, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    (out, err) = proc.communicate()
+    if not proc.returncode == 0:
+        print_error(out, err, 'Call to measurement.py failed.')
+
+def upload_to_benchview(python, coreclrRepo, benchviewPath, uploadToBenchview, benchviewGroup, runType, configuration, operatingSystem, etwCollection, optLevel, jitName, pgoOptimized, architecture):
+    measurementJson = os.path.join(os.getcwd(), 'measurement.json')
+    buildJson = os.path.join(coreclrRepo, 'build.json')
+    machinedataJson = os.path.join(coreclrRepo, 'machinedata.json')
+    submissionMetadataJson = os.path.join(coreclrRepo, 'submission-metadata.json')
+
+    if not os.path.isfile(measurementJson):
+        raise Exception('%s does not exist. There is no data to be uploaded.' % measurementJson)
+    if not os.path.isfile(buildJson):
+        raise Exception('%s does not exist. There is no data to be uploaded.' % buildJson)
+    if not os.path.isfile(machinedataJson):
+        raise Exception('%s does not exist. There is no data to be uploaded.' % machinedataJson)
+    if not os.path.isfile(submissionMetadataJson):
+        raise Exception('%s does not exist. There is no data to be uploaded.' % submissionMetadataJson)
+
+    run_args = [python,
+            os.path.join(benchviewPath, 'submission.py'),
+            measurementJson,
+            '--build',
+            buildJson,
+            '--machine-data',
+            machinedataJson,
+            '--metadata',
+            submissionMetadataJson,
+            '--group',
+            benchviewGroup,
+            '--type',
+            runType,
+            '--config-name',
+            configuration,
+            '--config',
+            'OS',
+            operatingSystem,
+            '--config',
+            'Profile',
+            etwCollection,
+            '--config',
+            'OptLevel',
+            optLevel,
+            '--config',
+            'JitName',
+            jitName,
+            '--config',
+            'PGO',
+            pgoOptimized,
+            '--architecture',
+            architecture,
+            '--machinepool',
+            'PerfSnake']
+    log('')
+    log(" ".join(run_args))
+
+    proc = subprocess.Popen(run_args, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    (out, err) = proc.communicate()
+
+    if not proc.returncode == 0:
+        print_error(out, err, 'Createing BenchView submission data failed.')
+
+    if uploadToBenchview:
+        run_args = [python,
+                os.path.join(benchviewPath, 'upload.py'),
+                'submission.json',
+                '--container',
+                'coreclr']
```

Comment:
This should be the container name that was passed on the command line

Reply:
I think I missed that option.

Reply:
Nope, we just didn't have it before. I can change it, but this was copied from the cmd.

Reply:
Sorry, you are correct. This was only ever used for CoreCLR